tests/unit/test_prompt_manager.py

```diff
@@ -0,0 +1,352 @@
+import os
+import shutil
+
+import pytest
+
+from opendevin.utils.prompt import PromptManager
+
+
+@pytest.fixture
+def prompt_dir(tmp_path):
+    # Copy contents from "agenthub/codeact_agent" to the temp directory
+    shutil.copytree('agenthub/codeact_agent', tmp_path, dirs_exist_ok=True)
+
+    # Return the temporary directory path
+    return tmp_path
+
+
+SAMPLE_AGENT_SKILLS_DOCS = """Sample agent skills documentation"""
+
+
+@pytest.fixture
+def agent_skills_docs():
+    return SAMPLE_AGENT_SKILLS_DOCS
+
+
+CODEACT_SYSTEM_PROMPT = f"""A chat between a curious user and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the user's questions.
```

Comment:
Can we put it into a single jinja2 file and reuse it here and in `system_prompt.j2`?

Otherwise, we need to modify two places whenever we tweak prompts (we already have regression testing for prompts in integration tests, so it's not necessary to do it again here).

Reply:
sounds good - i can simplify the test (not test for exact match) and have the integration tests to test for this

Reply:
done!